package.json

```diff
@@ -22,5 +24,11 @@
   "bugs": {
     "url": "https://github.com/NoroffFEU/css-frameworks-ca/issues"
   },
-  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme"
+  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme",
+  "devDependencies": {
+    "bootstrap": "^5.2.0",
```

Comment:
Bootstrap should be a dependency, not a devDependency:

`npm uninstall bootstrap`
`npm install bootstrap`

Reply:
started all over again: bootstrap is now a dependency